code/modules/food&drinks/kitchen machinery/food_cart.dm

```diff
@@ -0,0 +1,147 @@
+#define STORAGE_CAPACITY 30
+
+/obj/machinery/food_cart
+	name = "food cart"
+	desc = "New generation hot dog stand."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "foodcart"
+	density = 1
+	anchored = 0
+	use_power = 0
+	var/food_stored = 0
+	var/glasses = 0
+	var/portion = 10
+	var/selected_drink
+	var/list/stored_food = list()
+	flags = OPENCONTAINER | NOREACT
+	reagents = new()
+	var/obj/item/weapon/reagent_containers/mixer = new()
+
+/obj/machinery/food_cart/New()
+	..()
+	reagents.my_atom = src
+	mixer.name = "Mixer"
+	mixer.volume = 100
```

Comment:
This doesn't work. I think you'll need to set mixer.reagents.maximum_volume to 100 as well, since you're `New()`ing the mixer reagents as the default of 30. A less hacky solution would be making `reagent_container/New()` take a volume argument, and pass that to its `create_reagents()`